mobile_verifier/src/verifier.rs

```diff
@@ -139,17 +137,22 @@ impl VerifierDaemon {
         };
         metrics::gauge!("data_transfer_rewards_scale", scale);
 
-        for reward_share in
-            poc_rewards.into_radio_shares(&transfer_rewards, &scheduler.reward_period)
+        for (radio_reward_share, mobile_reward_share) in
+            poc_rewards.into_rewards(&transfer_rewards, &scheduler.reward_period)
         {
             self.radio_rewards
-                .write(reward_share, [])
+                .write(radio_reward_share, [])
+                .await?
+                // Await the returned one shot to ensure that we wrote the file
+                .await??;
+            self.mobile_rewards
+                .write(mobile_reward_share, [])
                 .await?
                 // Await the returned one shot to ensure that we wrote the file
                 .await??;
         }
 
-        let written_files = self.radio_rewards.commit().await?.await??;
```

Comment:
do these not _both_ need to be committed for as long as we're writing out the legacy `radio_reward_shares`?

Reply:
I changed `radio_reward_shares` to auto commit